db/schema.rb

```diff
@@ -691,6 +691,17 @@
     t.string "type", default: "Finance::Schedule::ECF"
   end
 
+  create_table "school_access_tokens", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
What is this? It doesn't seem related to the PR.

Reply:
Sry, migration from the other branch, removing now